Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Pacstall/pacstall](https://hosted.weblate.org/projects/pacstall/pacstall/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/pacstall/pacstall/horizontal-auto.svg)

oklopfer reviewed (changes requested):
Don't merge this until a new translation change is in to overwrite the broken commits

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" add-on in Weblate.

Translation: Pacstall/pacstall
Translate-URL: https://hosted.weblate.org/projects/pacstall/pacstall/

Vitor-RT committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 97.3% (218 of 224 strings)

Translation: Pacstall/pacstall
Translate-URL: https://hosted.weblate.org/projects/pacstall/pacstall/pt_BR/

oklopfer committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 97.3% (218 of 224 strings)

Translation: Pacstall/pacstall
Translate-URL: https://hosted.weblate.org/projects/pacstall/pacstall/pt_BR/
